packages/ra-ui-materialui/src/input/CheckboxGroupInput.tsx

```diff
@@ -80,7 +81,8 @@ import { LinearProgress } from '../layout';
  * The object passed as `options` props is passed to the material-ui <Checkbox> components
  */
 const CheckboxGroupInput: FunctionComponent<
-    ChoicesInputProps<CheckboxProps> & FormControlProps
+    ChoicesInputProps<CheckboxProps> &
+        FormControlProps & { classes?: ClassesOverride<typeof useStyles> }
```

Comment:
At this point, I think we should extract this into a `CheckboxGroupInputProps` interface and export it (don't forget the index file)

Reply:
Done